scripts/package.js

```diff
@@ -0,0 +1,86 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Please also add a `README.md` file with markdown for i) what the repo does (to create crx packages for extensions by Brave which are included with Brave), ii) What steps to perform to use it `yarn data-files-ad-block`, `package-ad-block`, and a description of the args for `package-ad-block`, mostly a more english version of what's in `package.js` for that.